src/lib/components/chat/ToolUpdate.svelte

```diff
@@ -64,7 +70,7 @@
 		>
 			<div
 				bind:this={loadingBarEl}
-				class="absolute -m-1 hidden h-full w-full rounded-l-lg bg-purple-500/5 transition-all dark:bg-purple-500/10"
+				class="absolute -m-1 hidden h-full w-[calc(100%+1rem)] rounded-lg bg-purple-500/5 transition-all dark:bg-purple-500/10"
```

Comment:
looked more like a progress bar without the rounded lg on the right imo (you could add an overflow-hidden on the parent instead).

Reply:
agreed, working on bringing it back but fighting with the padding :laughing:

Reply:
it was fine as it was imo :D even if it wasn't taking the full width